eng/tools/XplatPackageSigner/sign-packages.cmd

```diff
@@ -0,0 +1,15 @@
+@ECHO OFF
```

Comment:
I assume this script will be added as a build step in master's code signing build configuration. Is that correct?

Reply:
already setup and tested on this config: http://aspnetci/viewType.html?buildTypeId=CodeSignXplat